### ISSUE TITLE
Custom translations don't work

### ISSUE THREAD

**Issue by decabeza** [consuldemocracy/consuldemocracy#5703]
## Description

After the upgrade to release 2.2.0 custom translations don't work.

### Steps to reproduce

1. In a clean installation create the file `config/locals/custom/en/general.yml`
2. Add the content:
```
en:
  layouts:
    footer:
      consul: NEW AWESOME application
```
3. Run the application locally and the changes have no effect. 🥲

## Footer displayed (same original text)
![displayed](https://github.com/user-attachments/assets/262cc436-f6cd-4fe7-8038-e08c878f9529)

## Footer expected (with custom text)
![expected](https://github.com/user-attachments/assets/910744e5-1277-44f9-969a-25c0dd1ef4a7)

**Comment by javierm** [comment]
Hi, @decabeza :smile:.

Thanks for reporting! I think it was fixed in pull request #5660. As a workaround, you can add the following lines to your `application_custom.rb` file:

```ruby
   # TODO: Remove after upgrading to Consul Democracy 2.2.1 or later,
   # since that version already includes this code
   initializer :exclude_custom_locales_automatic_loading, before: :add_locales do
     paths.add "config/locales", glob: "**[^custom]*/*.{rb,yml}"
   end
```

**Comment by javierm** [comment]
I've added information about this behavior and the workaround to [version 2.2.0 release notes](https://github.com/consuldemocracy/consuldemocracy/releases/tag/2.2.0).
